### ISSUE TITLE
Text does not wrap when isScrollingEnabled is set to false

### ISSUE THREAD

**Issue by mattmaddux** [kenmueller/TextView#7]
If you write a long line with the following view...

```
struct TestView: View {
    
    @State var text: String = ""
    @State var isEditing: Bool = false
    
    var body: some View {
        TextView(text: $text, isEditing: $isEditing, isScrollingEnabled: false)
    }
    
}
```

...the text won't wrap and view keeps getting wider.

But if you just change it back to true...

```
struct TestView: View {
    
    @State var text: String = ""
    @State var isEditing: Bool = false
    
    var body: some View {
        TextView(text: $text, isEditing: $isEditing, isScrollingEnabled: true)
    }
    
}
```

...it works fine.

**Comment by kenmueller** [comment]
Try setting the width explicitly, does it still have that problem?

**Comment by mattmaddux** [comment]
Yeah, unfortunately it still doesn't wrap even if you set the width. You can see below I put a border just so you can see the shape of the view. It looks like the SwiftUI view is staying the appropriate width, but the UIView inside it is expanding.

![2020-02-26 10 47 43](https://user-images.githubusercontent.com/6632566/75367327-bfcd0300-5885-11ea-8405-d0fb21de55a3.gif)

**Comment by cblackcom** [comment]
Is it possible that the `isScrollingEnabled` property should actually be `isScrollEnabled` ?

**Comment by kenmueller** [comment]
No, I set the `isScrollEnabled` property on the `UITextView`, I just slightly changed the name for the constructor parameter.

**Comment by ninjudd** [comment]
This should fix it: https://stackoverflow.com/a/60278105
